joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 200  

left_wheel_velocity_controller:
    type: "velocity_controllers/JointVelocityController"
    joint: "base_left_wheel_joint"
    pid : {p: 100.0 , i: 0.0 1 , d: 10.0 }

    
right_wheel_velocity_controller:
    type: "velocity_controllers/JointVelocityController"
    joint: "base_right_wheel_joint"
    pid : {p: 100.0 , i: 0.0 1 , d: 10.0 }
